### PR TITLE
[dev-launcher] ui updates for landscape orientation

### PR DESCRIPTION
**Pull request by ajsmth**
# Why

Closes ENG-5789

I did a quick audit of styles for landscape orientation. A couple of screens had some buttons cut off / hard to reach

# How

Tweaks to padding and margin when devices are past a certain width

# Test Plan

Video: 

I tested on a iPhone 13 simulator and a variety of iPad sizes

https://user-images.githubusercontent.com/40680668/188729952-957689f9-47f9-4693-b9f0-8d1da11616c3.mp4

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [x] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by lukmccall — APPROVED** [review]
Looks good ;)
I have only one additional question did you test it on Android too?

**Comment by ajsmth** [comment]
Good call! Looks like I had to update the `AndroidManifest.xml` to allow for landscape orientation - the styles look good on Android as well.

File: packages/expo-dev-launcher/bundle/App.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import {
   SettingsFilledIcon,
 } from 'expo-dev-client-components';
 import * as React from 'react';
+import { useWindowDimensions } from 'react-native';
```
Comment (esamelson) [inline packages/expo-dev-launcher/bundle/App.tsx:10]:
```suggestion
```

**Review by esamelson — APPROVED** [review]
lgtm!

**Pull request merged**